### ISSUE TITLE
Warning: insecure environment read function 'getenv' used

### ISSUE THREAD

**Issue by csandanov** [drush-ops/drush#4148]
**Describe the bug**

The following warning shows up when a drush command executed:
```shell
$ drush version
D0808 03:22:55.490140185     906 env_linux.cc:71]            Warning: insecure environment read function 'getenv' used
 Drush version : 9.7.1
```

**To Reproduce**
Run any drush command

**Expected behavior**
No warning should show up

**Actual behavior**
A warning shows up

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 9.7.1
| Drupal version? | 8.7.6
| PHP version     | 7.3.8
| OS?             | Alpine Linux 3.10

**Comment by csandanov** [comment]
It seems it's related to php (or some of it extensions, grpc?) rather than to drush

**Comment by weitzman** [comment]
Are you able to try PHP 7.2 and see if warning goes away? I dont think this is a Drush error. Example https://github.com/grpc/grpc/issues/9509

**Comment by csandanov** [comment]
The issue was in the recent grpc extension update, should be fixed in upcoming 1.23.0 https://github.com/grpc/grpc/issues/19366
